geps/gep-713/index.md

```diff
@@ -1228,7 +1227,7 @@ Thus its status as EXPERIMENTAL DO NOT USE YET.
 
 Implementations MAY supply a Validating Admission Webhook that will return a
 WARNING message when an applied object is affected by some Policy, which may be 
-an inherited or indirect one.
+an inherited one.
```

Comment:
I think this meant to say "or direct". Could probably remove everything after "by some Policy" and get a simpler result.

Reply:
Okay, will do.

The "indirect" got me thinking of another scenario. A metaresource is a resource so it can be the target of another metaresource, right? So let's say I have a resource A which is an HTTPRoute. Then I have a metaresource (policy) B whose target is A. And I have another metaresource (policy?) C whose target is B. Is that possible? If yes, then C is indirectly or transitively attached to A. Let me know if my understanding is not correct.

Reply:
I think that's theoretically possible, but it has not been defined anywhere. I'd have to see a specific use case, but on the surface it seems like it would be pretty difficult for a user to follow.

Reply:
Agreed with all of the above, I think that striking the text after `by some Policy` is better.

Reply:
> Agreed with all of the above, I think that striking the text after `by some Policy` is better.

Done